code/player/Player.cs

```diff
@@ -10,13 +11,16 @@ public partial class TTTPlayer : Sandbox.Player
         public enum RoleType { None, Innocent, Detective, Traitor }
         public PlayerCorpse PlayerCorpse { get; set; }
 
-        [Net, Local]
+        // TODO: Make LOCAL, if it isn't this data gets networked to all players, could cause hackers.
+        // TODO: It currently isn't because all Networked information needs to be transfered to player corpse for inspecting bodies.
+        [Net]
         public RoleType Role { get; set; }
 
         [Net, Local]
         public int Credits { get; set; } = 0;
 
         private DamageInfo _lastDamageInfo;
+        private float _inspectCorpseDistance = 80f;
```

Comment:
Private vars with the prefix `_` or just lower case? I used the last one, but we could use the first style as well.

Reply:
Uhh whichever one we use, I'll check back previous code or ask in the discord.

Reply:
@saibotk @Histalek @TimGoll @AMastryukov @Westsaid @lebroomer  :D

Reply:
I don't really care but the underscore makes it really obvious.

Reply:
i don't mind, but please remember to change the .editorconfig accordingly

https://github.com/TTTReborn/ttt-reborn/blob/298dbdd352eb94dd0e4e29590a18b9c6701acfc7/.editorconfig#L68

Reply:
Seems like we go with lowercase, I'll do a refactor in another branch.